public/js/src/api.js

```diff
@@ -0,0 +1,53 @@
+const POLITICS_URL = `https://content.guardianapis.com/search?from-date=${yesterday()}&to-date=${today()}&q=politics&api-key=test&show-fields=all`
+
+const frontPage = new FrontPage();
+
+fetchArticles();
+
+function fetchArticles(url = POLITICS_URL) {
+  if (ENVIRONMENT === 'test') {
+    const results = '{"response":{"status":"ok","results":"Test OK"}}'
+    frontPage.articles = results;
+    return results;
+  }
+  fetch(url, {
+    method: 'GET',
+    mode: 'cors',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }).then(response => response.json())
+    .then(data => frontPage.articles = data.response.results);
+}
+
+function today() {
+  const today = new Date().toLocaleDateString();
```

Comment:
I never got beyond using the guardian test api - assuming the api needs you to put in a date range. this is very smart